doc/reference/stdio.md

```diff
@@ -155,7 +155,7 @@ Returns the peer address for a connected socket; this is undefined for ServerSoc
   option := fixnum
 ```
 Gets a socket option.
-See `:std/os/socket` for supported sockopts and their value types.
+See [`:std/os/socket`](sockets.md) for supported sockopts and their value types.
```

Comment:
Oh, so here you do the linking...

Reply:
yeah, I'll try to be more consistent about it.